helm/hpcc/values.yaml

```diff
@@ -183,6 +200,9 @@ esp:
   replicas: 1
   servicePort: 8010
   public: false
+  #resources:
+  #  cpu: "2"
```

Comment:
these are all a guess/example, but not sure really, but services having more than 1 cpu vs others, may be misleading.